tests/README.md

```diff
@@ -28,25 +28,17 @@ ssh on the libvirt domain and have Ansible installed on the host machine.
 ### Domain preparation
 
 
-1. Install domain, using `kickstarts/rhel_centos_7.cfg`
+1. Install domain, using provided kickstart
 
    To install the domain via graphical install, supply boot option:  
    `inst.ks=https://raw.githubusercontent.com/OpenSCAP/scap-security-guide/master/tests/kickstarts/rhel_centos_7.cfg`.  
    And the installation will be guided by the kickstart.
 
-   To install the domain via command line, see example command below:  
-   1. To install a CentOS VM:
-    ```bash
-    virt-install -n ssg-test-suite-centos -r 4096 --vcpus=4 \
-    --os-variant=rhel7 --accelerate \
-    --disk path=/var/lib/libvirt/images/ssg-test-suite-centos.qcow,size=50 \
-    -x "inst.ks=https://raw.githubusercontent.com/OpenSCAP/scap-security-guide/master/tests/kickstarts/rhel_centos_7.cfg" \
-    --location http://mirror.centos.org/centos/7/os/x86_64
-    ```
-   2. To setup a Fedora machine, change the following parameters:  
-     `--os-variant=fedora28`  
-     `--location https://dl.fedoraproject.org/pub/fedora/linux/releases/28/Everything/x86_64/os`  
-     NOTE: The Everything compose is necessary to be able to install `openscap-scanner` during kickstart installation.
+   To install the domain via command line, you can use script provided:
+   ```
+   ./install_vm.py --help
```

Comment:
If I run this command, it only shows a help and does not install anything.

Reply:
good point